packages/mdsvex/package.json

```diff
@@ -34,6 +34,7 @@
 		"@types/estree": "^0.0.45",
 		"@types/hast": "^2.3.1",
 		"@types/js-yaml": "^3.12.5",
+		"cpy-cli": "^3.1.1",
```

Comment:
I don't wanna bring in more deps just to copy a single file.

Reply:
Replaced with custom node script